### PR TITLE
Remove Code in navigator FS

### PR DESCRIPTION
**Pull request by gbalint**
**Description:**
Remove the `Code in navigator` feature switch.

- Removed duplicated version of tests which are already fixed for the true case
- Fixed the remaining failing tests.
- Added new tests

### CODE REVIEW

**Review by Rheeseyb — CHANGES_REQUESTED** [review]
PR looks good, but we might as well remove the feature switch entirely now

**Review by Rheeseyb — APPROVED** [review]

File: editor/src/components/canvas/controls/select-mode/select-mode.spec.browser2.tsx [hunk 1]
```diff
@@ -364,16 +364,17 @@ describe('Select Mode Clicking', () => {
...
-      '/CardList-Col',      // <- Single click
-      '/CardList-Card~~~1', // <- First *and* Second double click, as the Second is required to focus it
-    )
+    'sb' +                 // Skipped as it's the storyboard
+    '/scene-CardList' +    // Skipped because we skip over Scenes
+    '/CardList-instance' + // Skipped because we skip component children of Scenes
+    ':CardList-Root' +     // Skipped because we skip over root elements
+    '/CardList-Col',      // <- Single click
+    '/6d6',               // <- First double click selects the expression item
+    '/CardList-Card~~~1', // <- Second *and* third double click, as the Second is required to focus it
```
Comment (balazsbajorics) [inline editor/src/components/canvas/controls/select-mode/select-mode.spec.browser2.tsx:376]:
```suggestion
    '/CardList-Card~~~1', // <- Second *and* third double click, as the third is required to focus it
```

File: editor/src/components/canvas/controls/select-mode/select-mode.spec.browser2.tsx [hunk 2]
```diff
@@ -405,23 +406,27 @@ describe('Select Mode Clicking', () => {
...
-        '/Card-Row-Buttons',   // <- Third double click
-        '/Card-Button-3',      // <- Fourth double click
-      )
+      'sb' +                 // Skipped as it's the storyboard
+      '/scene-CardList' +    // Skipped because we skip over Scenes
+      '/CardList-instance' + // Skipped because we skip component children of Scenes
+      ':CardList-Root' +     // Skipped because we skip over root elements
+      '/CardList-Col',       // <- Single click
+      '/6d6',               // <- First double click selects the expression item
+      '/CardList-Card~~~1',  // <- Second *and* third double click, as the Second is required to focus it
```
Comment (balazsbajorics) [inline editor/src/components/canvas/controls/select-mode/select-mode.spec.browser2.tsx:425]:
```suggestion
      '/CardList-Card~~~1',  // <- Second *and* third double click, as the third is required to focus it
```

File: editor/src/components/canvas/controls/select-mode/select-mode.spec.browser2.tsx [hunk 3]
```diff
@@ -670,129 +679,16 @@ describe('Select Mode Double Clicking With Fragments', () => {
...
-      '/scene-CardList' +    // Skipped because we skip over Scenes
-      '/CardList-instance' + // Skipped because we skip component children of Scenes
-      ':CardList-Root' +     // Skipped because we skip over root elements
-      '/CardList-Col',      // <- Single click
-      '/6d6',               // <- First double click selects the expression item
-      '/CardList-Card~~~1', // <- Second *and* third double click, as the Second is required to focus it
-    )
-
-      const renderResult = await renderTestEditorWithCode(
-        TestProjectAlpineClimb,
```
Comment (balazsbajorics) [inline editor/src/components/canvas/controls/select-mode/select-mode.spec.browser2.tsx:688]: was it deliberate to change the test project from TestProjectAlpineClimb to TestProjectAlpineClimbWithFragments? am I reading the diff wrong?

File: editor/src/components/canvas/controls/select-mode/select-mode.spec.browser2.tsx [hunk 4]
```diff
@@ -872,16 +773,20 @@ describe('Select Mode Double Clicking With Fragments', () => {
...
     await doubleClick()
-    checkFocusedPath(renderResult, desiredPaths[1])
+    checkFocusedPath(renderResult, desiredPaths[2])
     checkSelectedPaths(renderResult, [desiredPaths[3]])
+
+    await doubleClick()
+    checkFocusedPath(renderResult, desiredPaths[2])
+    checkSelectedPaths(renderResult, [desiredPaths[4]])
   })
   // This test fails because there is a generated component there with a root fragment, which case is not supported
```
Comment (balazsbajorics) [inline editor/src/components/canvas/controls/select-mode/select-mode.spec.browser2.tsx:791]: is it possible this test could work now with these changes?

File: editor/src/components/canvas/ui-jsx-canvas-renderer/ui-jsx-canvas-element-renderer-utils.tsx [hunk 5]
```diff
@@ -97,18 +97,7 @@ export function createLookupRender(
       jsExpressionValue(generatedUID, emptyComments),
     )
 
-    // TODO BALAZS should this be here? or should the arbitrary block never have a template path with that last generated element?
-    const elementPathWithoutTheLastElementBecauseThatsAWeirdGeneratedUID = optionalMap(
```
Comment (balazsbajorics) [inline editor/src/components/canvas/ui-jsx-canvas-renderer/ui-jsx-canvas-element-renderer-utils.tsx:101]: yay!

**Review by balazsbajorics — APPROVED** [review]

File: editor/src/components/canvas/controls/select-mode/select-mode.spec.browser2.tsx (hunk 3 above)
Comment (gbalint) [reply editor/src/components/canvas/controls/select-mode/select-mode.spec.browser2.tsx:688]: I think the diff is hard to follow, there are two versions of these tests, both with `TestProjectAlpineClimb` and `TestProjectAlpineClimbWithFragments` (just like earlier)

File: editor/src/components/canvas/controls/select-mode/select-mode.spec.browser2.tsx (hunk 4 above)
Comment (gbalint) [reply editor/src/components/canvas/controls/select-mode/select-mode.spec.browser2.tsx:791]: No, this is caused by a limitation of `firstAncestorOrItselfWithValidElementPath`, and that is independent of the changes in this PR
